frontend/src/components/Recipes/Details/RecipeComments.tsx

```diff
@@ -0,0 +1,108 @@
+import { useRef, useState } from 'react'
+import { endpoint, RecipeCommentType, submitComment } from '../../../api/recipe'
+import { useData } from '../../../hooks/useData'
+
+import { RecipeComment } from './RecipeComment'
+import SkeletonCard from '../SkeletonCard'
+
+type RecipeCommentsProps = {
+    recipeId: string | undefined
+}
+
+export function RecipeComments({ recipeId }: RecipeCommentsProps) {
+    const formRef = useRef<HTMLFormElement>(null)
```

Comment:
Personally, I would have used a controlled form, have you chosen an uncontrolled approach for any specific reason?

Reply:
Want to try having 2 different form types when introducing react-hook-forms.